src/components/FooterItem/FooterItem.tsx

```diff
@@ -8,7 +8,7 @@ import './FooterItem.scss';
 
 const b = block('footer-item');
 
-export interface FooterItemProps extends ItemProps {
+export interface FooterItemProps extends Omit<ItemProps, 'onItemClick'> {
```

Comment:
What is the meaning of this change?

Reply:
Without this change we have two ways to set onItemClick handler: ItemProps.onItemClick and ItemProps.item.onItemClick. And I don't think it is a good idea to have such variety. So this change leaves only one backward compatible way to set the handler.

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys